gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationViewModel.java

```diff
@@ -89,6 +90,9 @@
   /** Toolbar title, based on whether user is adding new or editing existing observation. */
   private final MutableLiveData<String> toolbarTitle = new MutableLiveData<>();
 
+  /** Captured / selected photo. */
+  private final MutableLiveData<Map<Field, String>> photoUpdated = new MutableLiveData<>();
```

Comment:
By convention, we've been using plural nouns for streams, i.e. `photoUpdates`; it can be thought of a shorthand for "a stream of updates to photo fields".

Reply:
Done. Ty